index.js

```diff
@@ -150,8 +154,12 @@ module.exports = {
       nemo.driver.flow_.on('uncaughtException',function(exception){
         var filename = 'ScreenShot_onException-' + process.pid + '-' + new Date().getTime();
         var screenShotFileName = path.resolve(screenShotPath, filename);
-        nemo.screenshot.snap(screenShotFileName).then(function(){
-        });
+        try{
+          nemo.screenshot.snap(screenShotFileName).then(function(){
```

Comment:
You are right. This function is unnecessary. Will remove it when I update the pull request.